ananas/src/main/java/iamutkarshtiwari/github/io/ananas/editimage/ImageEditorIntentBuilder.kt

```diff
@@ -141,5 +141,37 @@ class ImageEditorIntentBuilder @JvmOverloads constructor(private val context: Co
         const val FORCE_PORTRAIT = "force_portrait"
         const val EDITOR_TITLE = "editor_title"
         const val SUPPORT_ACTION_BAR_VISIBILITY = "support_action_bar_visibility"
+
+        fun getImageEditorIntent(context: Context, imageLocation: Uri, outputPath: String): Intent {
+            return ImageEditorIntentBuilder(context, imageLocation, outputPath)
+                    .withAddText() // Add the features you need
+                    .withPaintFeature()
+                    .withFilterFeature()
+                    .withRotateFeature()
+                    .withCropFeature()
+                    .withBrightnessFeature()
+                    .withSaturationFeature()
+                    .withBeautyFeature()
+                    .withStickerFeature()
+                    .forcePortrait(true) // Add this to force portrait mode (It's set to false by default)
```

Comment:
why do we need this?

Reply:
If force portrait is not set the orientation will be locked for editor activity based on what orientation the device was. If user triggers image editor while the device is in landscape, then. image editor will be in landscape too.

Reply:
ok, in this case then let's keep the forcePortrait(true) as you initially did